tools/packaging/static-build/tools/Dockerfile

```diff
@@ -2,35 +2,49 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM alpine:3.18
+FROM ubuntu:20.04
```

Comment:
@Sumynwa if it is necessary to switch, do we want to consider a more recent version?

Reply:
We've had this discussion, and switching to a newer version was not something fruitful.

I think we're good with the switch, @cmaf.